pkg/sentry/socket/hostinet/socket.go

```diff
@@ -41,6 +41,10 @@ const (
 	// sizeofSockaddr is the size in bytes of the largest sockaddr type
 	// supported by this package.
 	sizeofSockaddr = syscall.SizeofSockaddrInet6 // sizeof(sockaddr_in6) > sizeof(sockaddr_in)
+
+	sockAddrInetSize = uint32(syscall.SizeofSockaddrInet4)
```

Comment:
Was trying to avoid too many type conversion where it's used.

I think commit 7bfad8ebb6ce ("Return a well-defined socket address type from socket funtions") already fixed this issue, will drop this PR.